code/client/munkiimport

```diff
@@ -691,12 +691,17 @@ def cleanupAndExit(exitcode):
     exit(exitcode or result)
 
 
-def pref(prefname):
-    """Returns a preference for prefname"""
+def pref(prefname, value=None):
+    """Returns or sets preference for prefname"""
     try:
         _prefs = FoundationPlist.readPlist(PREFSPATH)
     except FoundationPlist.NSPropertyListSerializationException:
         return None
+
+    if value:
+        _prefs[prefname] = value
+        return FoundationPlist.writePlist(_prefs, PREFSPATH)
+
     if prefname in _prefs:
```

Comment:
Yuck. Sometimes the function returns a value and sometimes it returns nothing. Seems like this should be two different functions -- one that gets, and one that sets.